NDTensors/src/lib/GradedAxes/src/fusion.jl

```diff
@@ -0,0 +1,110 @@
+using BlockArrays: BlockedUnitRange
+
+# TODO: Decide what to do about `dual`. Should there just
+# be a version in `Sectors`?
+## function dual end
```

Comment:
See discussion, this question boils down to choosing if a GradedUnitRange can act as a reducible representation / vector space.

Reply:
Agreed, this PR was deliberately designed to be more narrow, and not address the question around `dual` yet.